de.dlr.sc.virsat.model.concept.test/src/de/dlr/sc/virsat/model/concept/generator/validator/GenerateValidatorTest.xtend

```diff
@@ -45,7 +45,7 @@ class GenerateValidatorTest {
     def void testCreateConcreteClassFileName() {
     	val fileName = validatorGenerator.createConcreteClassFileName(concept, concept)
     	
-    	val expectedFileName = testConceptName + "/validator/StructuralElementInstanceValidator.java"
+    	val expectedFileName = testConceptName + "/validator/testConceptValidator.java"

```

Comment:
By convention class names should start with a capital letter "TestConceptValidator.java" same as the class.